projects/default/controllers/ros/RosCamera.cpp

```diff
@@ -107,7 +113,40 @@ void RosCamera::publishValue(ros::Publisher publisher) {
   publisher.publish(image);
 }
 
+sensor_msgs::CameraInfo RosCamera::createCameraInfoMessage() {
+  sensor_msgs::CameraInfo info;
+  info.header.stamp = ros::Time::now();
+  info.header.frame_id = mFrameIdPrefix + RosDevice::fixedDeviceName();
+
+  const double width = mCamera->getWidth();
+  const double height = mCamera->getHeight();
+  info.width = width;
+  info.height = height;
+
+  const double horizontalFov = mCamera->getFov();
+  double focalLength = width / (2.0 * tan(horizontalFov / 2.0));
```

Comment:
I believe this one could be const as well.
```suggestion
  const double focalLength = width / (2.0 * tan(horizontalFov / 2.0));
```